pkg/libsf/session.go

```diff
@@ -30,3 +35,41 @@ func (s Session) AccessExpired() bool {
 func (s Session) RefreshExpired() bool {
 	return !s.Defined() || time.Now().After(s.RefreshExpiration)
 }
+
+type sessionTime time.Time
+
+func (t *sessionTime) UnmarshalJSON(s []byte) error {
+	r := strings.Replace(string(s), `"`, ``, -1)
+	tt, err := time.Parse(time.RFC3339, r)
+	if err != nil {
+		q, err := strconv.ParseInt(string(s), 10, 64)
+		if err != nil {
+			return err
+		}
+		*(*time.Time)(t) = time.Unix(q/1000, 0)
```

Comment:
I'm not sure it's very useful, but since the timestamp is provided in millisecond, we should use the most accurate possible value: `time.Unix(0, q*1_000_000)`